Makefile

```diff
@@ -1,5 +1,7 @@
 all: install_requirements configure_host launch_mgmt_cluster verify
 
+nodep: configure_host launch_mgmt_cluster verify
```

Comment:
This is right now the same as the ci_run but I intend to keep them separate because they might diverge in the future.